pkg/server/test/list_objects.go

```diff
@@ -490,19 +491,19 @@ condition condition1(x: int) {
 				opts = append(opts, commands.WithListObjectsDeadline(test.listObjectsDeadline))
 			}
 
-			checkResolver, closer := graph.NewLayeredCheckResolver(
-				[]graph.LocalCheckerOption{
-					graph.WithResolveNodeBreadthLimit(100),
-					graph.WithMaxConcurrentReads(30),
-				},
-				test.useCheckCache,
-				false,
-				[]graph.CachedCheckResolverOpt{
-					graph.WithMaxCacheSize(100),
-					graph.WithCacheTTL(10 * time.Second),
-				},
-				[]graph.DispatchThrottlingCheckResolverOpt{},
-			)
+			localCheckOpts := []graph.LocalCheckerOption{
+				graph.WithResolveNodeBreadthLimit(100),
+				graph.WithMaxConcurrentReads(30),
+			}
+			cacheOpts := []graph.CachedCheckResolverOpt{
+				graph.WithMaxCacheSize(100),
+				graph.WithCacheTTL(10 * time.Second),
+			}
```

Comment:
Is the new code equivalent to the old code? You aren't using `test.useCheckCache` anymore.

The fact that no tests failed makes me think we may have other issues too. 🤔 

In any case, I don't think we should have that flag, see https://github.com/openfga/openfga/issues/1409

Reply:
Resolving this in favour of https://github.com/openfga/openfga/issues/1409. Will be checked when the above issue is closed.